docker.local/config/conductor.config.yaml

```diff
@@ -20,6 +20,9 @@ aggregate_base_url: "http://localhost:9081/v2"
 
 sharder1_base_url: "http://localhost:7171/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7"
 
+default_hardfork : {
+    "Name" : "ares"
```

Comment:
This is incorrect yaml format. Should be `name: "ares"`

Reply:
Should be 
```
name: "..."
round: "..."

Reply:
Still not resolved. Should be

```yaml
default_hardfork:
  name: ...
  round: ...
```
Also no need to mention round since default HF will be applied at round 1.